docs/dev/ci-reference.adoc

```diff
@@ -40,7 +40,14 @@ If you need to run `odo` integration tests against a specific version of Openshi
         - odo logout
 ----
 
-== Running integration tests on Prow
+=== Test on Kubernetes/ Openshift 3.11
+
+Under travis kubernetes and openshift 3.11 test are run.
```

Comment:
We do have some content and plan ready to run minikube/minishift tests on PSI. Refer https://github.com/openshift/odo/issues/4287 for more content. I think we can add more content towards PSI and remove travis completely.

Reply:
Should we add the kubernetes and minishift one in this PR? or create another PR once the complete setup for kubernetes and minishift is completed. As it will reduce the work if we need to make some changes.

Reply:
> Should we add the kubernetes and minishift one in this PR?

Adding kubernetes and minishift for psi would be covered in another pr I think. I suggest to add a comment saying its in WIP and will be updated once done. So that we don't miss adding it further. However talking about travis doesn't make sense to me as currently we are not running single tests using travis.

Reply:
> I suggest to add a comment saying its in WIP
Are you asking me to add comment in the doc or on the issue/PR ?
other then that all of the point make sense.

Reply:
create an issue for that and link it here would make sense.

Reply:
Understood!